scripts/v6/prime/15-Breakdowns_SCOAClassification.csv

```diff
@@ -0,0 +1,11 @@
+Id,SCOAId,ParentSCOAId,DefinitionDescription,SCOAFile,SCOAAccount,SCOALevel,ExcelRowNumber,ShortDescription,VATStatus,BreakDownAllowed,Principle,ApplicableTo,bPostingLevel,PostingLevelDescription,AccountNumber,AccountNumberPrefix,AccountNumber1,AccountNumber2,AccountNumber3,AccountNumber4,AccountNumber5,AccountNumber6,AccountNumber7,AccountNumber8,AccountNumber9,AccountNumber10,AccountNumber11,AccountNumber12,GFSCode,Path,SCOASegment,Linkable,ROWID,IsBreakdown,IsSelectable,SCOAVersion
```

Comment:
I've made slight changes to the file naming convention: Everything between the last "-" and the extension ".csv" is the table name. Do you thing you can change the underscore to a dash?

Reply:
👍